packages/react-sdk/docusaurus/docs/React/03-ui-components/utility/call-stats.mdx

```diff
@@ -3,8 +3,28 @@ id: call-statistics
 title: Call statistics
 ---
 
-:::warning
+import CallStats from '../../assets/03-ui-components/call-stats.png';
 
-TODO: write about: https://github.com/GetStream/stream-video-js/tree/main/packages/react-sdk/src/components/CallStats
+The SDK's default component to display statistics about a call is called `CallStats`. It displays:
 
-:::
+- **Call Latency** graph that outlines average round trip time (in ms).
+- **Call Performance** section that displays the below listed call statistics
+
+<img src={CallStats} alt="Panel displaying call statistics" width={300}/>
+
+### The call statistics displayed:
+
+- **Region** - the SFU we are connected to
```

Comment:
I think SFU is an internal word, I don't think our integrators would necessarily know what it stand for, I'd just use server here

Reply:
replaced the use of SFU with server

Reply:
Thanks